numpy/lib/function_base.py

```diff
@@ -1027,9 +1027,11 @@ def average(a, axis=None, weights=None, returned=False):
     a : array_like
         Array containing data to be averaged. If `a` is not an array, a
         conversion is attempted.
-    axis : int, optional
-        Axis along which to average `a`. If `None`, averaging is done over
-        the flattened array.
+    axis : int or tuple of ints, optional
+        Axis or axes along which to average `a`. If `None`, averaging is done
+        over the flattened array.
+        If this is a tuple of ints, averaging is performed over multiple axes,
```

Comment:
Should be a single paragraph, i.e., combined with preceding line. I'd cut and paste from the corresponding part of the `sum` docstring with relevant changes to have a more complete explanation.